<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MO-Dolly" failures="1" tests="2" name="Test" time="12.684" errors="0" timestamp="2023-05-04T22:30:30 GMT-04:00">
  <testcase classname="TestCases.VerifySignUpPage" name="signup" time="5.932"/>
  <testcase classname="TestCases.VerifyLogin" name="login" time="0.425">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	ERROR expected [Error in Login] but found [Please check your username and password. If you still can&amp;apos;t log in, contact your Salesforce administrator.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
ERROR expected [Error in Login] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]
at TestCases.VerifyLogin.login(VerifyLogin.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Test -->
